viewer/src/public/RevealManager.ts

```diff
@@ -8,16 +8,36 @@ import { CadSectorParser } from '@/dataModels/cad/internal/sector/CadSectorParse
 import { SimpleAndDetailedToSector3D } from '@/dataModels/cad/internal/sector/SimpleAndDetailedToSector3D';
 import { CachedRepository } from '@/dataModels/cad/internal/sector/CachedRepository';
 import { MaterialManager } from '@/dataModels/cad/internal/MaterialManager';
+import { CadManager } from '@/dataModels/cad/internal/CadManager';
+import { CadModelMetadataRepository } from '@/dataModels/cad/internal/CadModelMetadataRepository';
+import { DefaultCadTransformation } from '@/dataModels/cad/internal/DefaultCadTransformation';
+import { CadMetadataParser } from '@/dataModels/cad/internal/CadMetadataParser';
+import { CogniteClient3dExtensions } from '@/utilities/networking/CogniteClient3dExtensions';
+import { CadModelFactory } from '@/dataModels/cad/internal/CadModelFactory';
+import { CadModelUpdateHandler } from '@/dataModels/cad/internal/CadModelUpdateHandler';
+import { ProximitySectorCuller } from '@/dataModels/cad/internal/sector/culling/ProximitySectorCuller';
 
 // First iteration of a RevealManager. Currently tailored to examples but should be tailored to external usecase.
 // Should move to example-helpers.ts as a function without extending
+// TODO: j-bjorne 15-05-2020: create a function that creates a default CadManager and place it in a util class?
 export class RevealManager extends RevealManagerBase {
-  constructor(client: CogniteClient, onUpdatedCallback: () => void, options?: RevealOptions) {
+  constructor(client: CogniteClient, options?: RevealOptions) {
```

Comment:
Nice to get rid of the onUpdatedCallback! Should introduce event as alternative to polling, but not the most pressing issue.